packages/material-ui/src/Chip/Chip.js

```diff
@@ -242,6 +248,7 @@ Chip.propTypes = {
 };
 
 Chip.defaultProps = {
+  clickable: false,
```

Comment:
Should the `clickable` property have a higher priority than the `onClick` detection logic?

Reply:
I feel clickable should have low priority than OnClick. The output of above will still be clickable as per the below code: `{ [classes.clickable]: onClick || clickable },` if onclick is present, it will be clickable at any cost. Only when onClick is not present, clickable comes into the picture.

Reply:
Ah, I see. So being able to set `clickable={somethingFalse}` when a Chip has an `onClick()` handler but other conditions mean that it won't act on the click?

Reply:
@oliviertassinari  I feel  defaultProps.clickable = null also would work fine. still false makes better sense, as we have kept props type as boolean. Also, in terms of functionality, nothing changes in either way

Reply:
@mbrookes Couldn't understand your question clearly. But to explain, if we have 'onClick' hander, 'clickable' doesn't affect and the chip is always clickable. when 'onClick' is not present, then the value of 'clickable' determine if the chip is clickable

Reply:
@vilvaathibanpb I was just clarifying with @oliviertassinari if I understood correctly the benefit of `defaultProps.clickable = null` combined with clickable having priority.

Reply:
> What do you see as the main benefit for null as the default?

The advantage is that it's allowing people to control the clickable state, but still, being able to rely on the onClick detection logic if they wish. But I'm fine either way. The current approach is simple too.